src/Trend/Trend.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './Trend.css';
+import PropTypes from 'prop-types';
+
+import ColorTrend from './ColorTrend';
+
+const Trend = (props) => {
+  let items = [];
+  if (props.type === 'color') {
+    props.favorites.forEach(piece => {
+      if (piece.colorcount >= 1) {
+        piece.colors.forEach(color => {
+          items.push(color.color)
+        })
+      }
+    })
+  } else if (props.type === 'people') {
+    props.favorites.forEach(piece => {
+      if (piece.peoplecount >= 1) {
+        piece.people.forEach(person => {
+          items.push(person.displayname)
+        })
+      }
+    })
+  } else {
+    props.favorites.forEach(piece => {
+      if (piece[props.type]) {
+        items.push(piece[props.type])
+      }
+    });
+  }
+  let itemsFreq = items.reduce((freq, item) => {
+    if (!freq[item]) {
+      freq[item] = 1
+    } else {
+      freq[item]++
+    }
+    return freq;
+  }, {})
+  let sorted = Object.entries(itemsFreq).sort((a, b) => b[1] - a[1]);
```

Comment:
Possibly find a way to move the sorting stuff somewhere else? Seems fine here, but could be a method in App maybe?